### ISSUE TITLE
Events for changes in local modules

### ISSUE THREAD

**Issue by jameslibscie** [p2pcommons/sdk-js#120]
**User story**
As a content creator
I want changes I make to my content to appear right away in the app
so that I don't have to manually refresh or re-open the app

**Acceptance criteria**
- [ ] Event for new version of a module
- [ ] Event for updated metadata of a module
- [ ] Event for changed files within a module

@juliangruber Do these events make sense? I kinda had a guess at them. Could you adjust the spec if needed and then replace `needs spec` with `ready to start`?

**Comment by juliangruber** [comment]
> * [ ] Event for new version of a module

:+1: This will happen after a file was changed, which we don't control in the app.

> * [ ]  Event for updated metadata of a module

We don't need this before the daemon, as currently while Desktop is open only Desktop can modify metadata.

> * [ ]  Event for changed files within a module

I would merge this with the first event, because this will also currently always cause a version update.

From the desktop perspective, it's currently enough to get one event for when the version (of a non-published module) was updated.

**Comment by jameslibscie** [comment]
Makes sense @juliangruber. I'm thinking that maybe later we might want to handle these things differently depending on our version control implementation, but we can deal with that then.

**Comment by chartgerink** [comment]
@jameslibscie how is this the same and/or different from #187?

**Comment by jameslibscie** [comment]
This watches for local changes made outside the SDK, while #187 is about keeping remote modules updated.
